pkg/upgrade/job/job.go

```diff
@@ -16,11 +17,12 @@ import (
 )
 
 const (
-	defaultBackoffLimit          = int32(2)
-	defaultActiveDeadlineSeconds = int64(600)
-	defaultPrivileged            = true
-	defaultKubectlImage          = "rancher/kubectl:1.17.0"
-	defaultImagePullPolicy       = corev1.PullIfNotPresent
+	defaultBackoffLimit            = int32(2)
+	defaultActiveDeadlineSeconds   = int64(600)
+	defaultPrivileged              = true
+	defaultKubectlImage            = "rancher/kubectl:1.17.0"
```

Comment:
ENV var here also? Might need to drop this down to upgrade a really old cluster

Reply:
that's the only one i have built and the client that system-upgrade-controller matches this so it's kine of moot. if the user needs a lower support version they are on their own (or if they ask nicely, maybe)

Reply:
Also, this is the default that gets used as a fallback. The provided manifest and hence the rancher/system-charts entry for SUC use a value from a configmap that overrides all of these defaults.